readme.md

```diff
@@ -111,6 +111,34 @@ cuts out back and forth of correction had someone else written it.
 See: https://github.com/dwyl/process-handbook/issues/61, linked to #5
 
 
+## Glossary of Key Terms
+
+- **Scrum Master**: tracks team progress, liaises with product owner on
+requirements, lends expertise on managing end to end projects, ensures features
+are tested when delivered and manages deployments in conjunction with technical
+architect and the team.
+- **Product backlog:** a prioritised features / to do list for the product found
+on github. See our ['What is the backlog?' section](#what-is-the-backlog?) for
+more detail.
+- **Sprint Backlog**: the backlog that is aiming for completion within a given
+sprint period.
+- **Sprint**: a 1-3 week working lifecycle of planning, building, presentation and
+reflection. We tend to work in 2 week (10 working day) sprints.
+- **UI / User Interface**: how the site looks from a design perspective.
+- **User testing:** interviews, focus groups and other forms of research conducted
+with potential/actual users of your product to give feedback and provide insight
+into what the user's needs are. See
+https://github.com/dwyl/learn-user-experience-testing to learn more about user
+testing.
+- **Milestone**:  a visual way to group issues in github, at dwyl we create one
+milestone per sprint so we have somewhere to see all of the issues which we
+estimate to complete by the end of the sprint. See
+https://github.com/dwyl/github-reference for more.
```

Comment:
There's apparently nothing on the github-reference repo on milestones!!!

Reply:
There's just an issue: https://github.com/dwyl/github-reference/issues/13